python/packages/nisar/workflows/baseline.py

```diff
@@ -355,57 +355,65 @@ def compute_baseline(ref_rngs,
     meta_rows, meta_cols = ref_rngs.shape
 
     # Initialize output arrays
-    perp_baseline_array = np.zeros([meta_rows, meta_cols])
-    par_baseline_array = np.zeros([meta_rows, meta_cols])
+    par_baseline_array = np.full((meta_rows, meta_cols),
+                                 np.nan, dtype=np.float32)
+    perp_baseline_array = np.full((meta_rows, meta_cols),
+                                  np.nan, dtype=np.float32)
 
     for row_ind in range(meta_rows):
         for col_ind in range(meta_cols):
```

Comment:
```suggestion
    # Initialize output arrays
    par_baseline_array = np.full(
        ref_rngs.shape, np.nan, dtype=np.float32
    )
    perp_baseline_array = np.full(
        ref_rngs.shape, np.nan, dtype=np.float32
    )

    for row_ind, col_ind in np.ndindex(ref_rngs.shape):
```

I recently learned that nested for loops over arrays like this can be flattened in this manner. Saves some bookkeeping and line space, and keeps it clear at a glance that all of these structures are the same dimensions as `ref_rngs`. Optional for this PR but something useful for the future!

Reply:
Good point. Thanks @Tyler-g-hudson I changed the lines as you suggested.